### ISSUE TITLE
FeatureRequest - Make it possible to change pool in console

### ISSUE THREAD

**Issue by EUViking** [fireice-uk/xmr-stak#999]
I belong to the ppl who runs miners via Linux/screen, so we have full control at any time. I had a problem today, where the pool responded with 'Duplicate Share' and then the rest of my shares where Rejected (forever?). The only solution, my patience-span is max a few minutes, was to restart, but as this is a 12 GPU rig restarts is not always an easy operation. Hence the need to re-login or switch pool from console:
a) Use single digits 1-9 to represent the pools (as in the config).
b) Pressing the same digit as an already active pool, causes a logout and a subsequent login.
c) Pressing another digit than the active pool, moves mining on that pool, as if the active pool no longer was reachable (maybe should cause a logout to the active pool and the try a fast login).

or just <whatever> letter to toggle around in the pool-list, as whatever logic applies to what happens when the active pool becomes unavailable.

And then there is a question is if you get a 'Your IP is Banned' (repeatedly) from the pool, if that initself should not trigger the same actions as when the Active Pool is no longer reachable?

**Comment by khobbs19** [comment]
Would love to see some kind of pool config.txt reload either from the keyboard input or if config.txt has changed

**Comment by DrStein99** [comment]
I agree.  I would like to change the pool other settings during runtime as well.
